doc/source/whatsnew/v1.0.0.rst

```diff
@@ -201,7 +201,7 @@ ExtensionArray
 Other
 ^^^^^
 - Trying to set the ``display.precision``, ``display.max_rows`` or ``display.max_columns`` using :meth:`set_option` to anything but a ``None`` or a positive int will raise a ``ValueError`` (:issue:`23348`)
-
+- Bug in pandas io formats style : fixed function default_display_func because wasn`t displaying decimals digits as supposed (:issue:`13257`)
```

Comment:
refer to the method on styler. :meth:\`...\` 

"wasn't" should use an apostrophe, not a backtick.

Reply:
My bad...
fixed